app/assets/javascripts/costs/editinplace.js

```diff
@@ -48,21 +48,21 @@ function edit(obj, name, obj_value) {
   var value = parsed[0];
   var currency = parsed[1];
 
-  var button = '<span id="'+obj.id+'_cancel" class="form--field-affix -transparent"><input id="'+obj.id+'_cancel" type="image" '+ _cancelButtonAttributes  +' /> </span>';
-  var span = '<span id="'+obj.id+'_editor" class="form--text-field-container">';
+  var form_start = '<section class="form--section" id="'+obj.id+'_section"><div class="form--field"><div class="form--field-container">';
```

Comment:
Line is too long.<br>Identifier 'form_start' is not in camel case.<br>Mixed double and single quotes.